code/modules/surgery/organs/augments_eyes.dm

```diff
@@ -31,6 +31,12 @@
 		REMOVE_TRAIT(eye_owner, HUD_trait, ORGAN_TRAIT)
 	..()
 
+/obj/item/organ/internal/cyberimp/eyes/hud/civ
```

Comment:
with this one too, i can just get a sechud implant

Reply:
And again, some security players will have an issue with it. Additionally, I have plans to make this available as a loadout item downstream.

Reply:
i agree, good for downstream.